packages/grafana-eslint-rules/README.md

```diff
@@ -12,6 +12,12 @@ Previously we hijacked the aria-label property to use as E2E selectors as an att
 
 Now, we prefer using data-testid for E2E selectors.
 
+### `@grafana/no-border-radius-literal`
+
+Check if border-radius theme tokens are used.
+
+To improve the consistency across Grafana we encourage devs to use tokens instead of custom values. In this case, we want the `borderRadius` to use the appropiate token such as `theme.shape.borderRadius()` or `theme.shape.radius.circle`.
+
```

Comment:
I'm missing the pill shape here. Is there a reason why it is not mentioned at all?

Reply:
Updated! :)